src/gen/README.md

```diff
@@ -0,0 +1,15 @@
+
+# Installation
+
+- Install maven
```

Comment:
Why not just create a bash file that would do everything? Also, maybe we should used docker container here so we'd have reproducible builds.

Reply:
done